### ISSUE TITLE
Database management commands fail if PGDATABASE set in environment

### ISSUE THREAD

**Issue by fiadliel** [theseus-rs/postgresql-embedded#75]
I'm not sure if you want to ascribe this as "fix your environment", but if `PGDATABASE` is set in the environment (to anything other than `postgres`), the database creation commands fail, as psql connects to a database that doesn't exist.

```
Error: Command error: stdout=; stderr=psql: error: connection to server at "localhost" (::1), port 58453 failed: FATAL:  database "xxx" does not exist
```

One option might be to explicitly set the database to `postgres` when running database management commands.

**Comment by brianheineman** [comment]
Hello @fiadliel, thank you for reporting this issue.  While PostgreSQL is running in another process, the aim of this project is to try to provide an "embedded-like" experience for users.  I made some changes in #76 that should address this problem and added a [regression test](https://github.com/theseus-rs/postgresql-embedded/blob/main/postgresql_embedded/tests/environment_variables.rs#L9) to try to reproduce/verify the fix.

**Comment by brianheineman** [comment]
The changes for #76 have been released in [0.10.2](https://github.com/theseus-rs/postgresql-embedded/releases/tag/v0.10.2)

**Comment by fiadliel** [comment]
Fixed in my test! (the scenario was that `PGDATABASE` was used when running programs interactively, but I was investigating using this library to set up some integration tests). Very quick fix, thanks a lot :)
